examples/ini_example.c

```diff
@@ -40,5 +40,7 @@ int main(int argc, char* argv[])
     }
     printf("Config loaded from 'test.ini': version=%d, name=%s, email=%s\n",
         config.version, config.name, config.email);
+    free((void *) config.name);
+    free((void *) config.email);
```

Comment:
Couple of nits:

* Can you please put a blank lines before and after the free block, i.e., before line 43 and after line 44, so they're kind of a standalone unit?
* Is the `(void *)` cast to avoid const warnings? If so, can you please format it without additional whitespace for consistency with [this style](https://github.com/benhoyt/inih/blob/535a7f7aef7cf1034a963d07806ea0a2a29c4e90/examples/ini_example.c#L18): `free((void*)config.name)`

Reply:
Thanks for the review.

>     * Can you please put a blank lines before and after the free block, i.e., before line 43 and after line 44, so they're kind of a standalone unit?

done 

> 
>     * Is the `(void *)` cast to avoid const warnings? If so, can you please format it without additional whitespace for consistency with [this style](https://github.com/benhoyt/inih/blob/535a7f7aef7cf1034a963d07806ea0a2a29c4e90/examples/ini_example.c#L18): `free((void*)config.name)`

Yes it is for avoiding the `const` warnings.
done.